src/app/plan/duck/types.ts

```diff
@@ -0,0 +1,13 @@
+const FETCH_DATA = 'FETCH_DATA';
```

Comment:
This file shouldn't exist; should use the Types generated from the reduxsauce actions file.